plutus-core/plutus-core/test/Pretty/Golden/Readable/Examples/Shad/mkShad.golden

```diff
@@ -1 +1,6 @@
-/\a -> iwrap (\(rec :: * -> *) -> \i -> a -> (all (a :: * -> *). a i -> integer)) a (\(x : a) -> /\(f :: * -> *) -> \(y : f a) -> 0)
\ No newline at end of file
+/\a ->
+  iwrap (\(rec :: * -> *)
+          i ->
+           a -> (all (a :: * -> *). a i -> integer)) a (\(x : a) ->
+                                                          /\f :: * -> * ->
+                                                            \(y : f a) -> 0)
```

Comment:
Borderline unreadable. We should render `iwrap` the way we render function application.

Reply:
I agree, I think this is the case for everything with multiple children!